### ISSUE TITLE
Silence journald rotation message in dmesg

### ISSUE THREAD

**Issue by plomari** [systemd/systemd#27258]
### Component

journalctl

### Is your feature request related to a problem? Please describe

I get this message fairly often in dmesg output:
```
systemd-journald[479]: Retention time reached, rotating.
```
I wish for systemd not to print such unimportant messages to the kernel log.

### Describe the solution you'd like

An option to disable this specific message.

### Describe alternatives you've considered

Making the log rotation period higher, but I specifically set it to a low value.

Writing a script to filter out this message, I'd prefer not to.

### The systemd version you checked that didn't have the feature you are asking for

252

**Comment by manio** [comment]
I have similar problem after upgrading debian's systemd:
`systemd:amd64 (247.3-7+deb11u2, 252.17-1~deb12u1)`

After this `systemd-journald` started spamming my dmesg with:
```
[Thu Oct 12 16:29:58 2023] systemd-journald[1457029]: Data hash table of /var/log/journal/XXXX/system.journal has a fill level at 75.0 (174764 of 233016 items, 67108864 file size, 383 bytes per hash table item), suggesting rotation.
[Thu Oct 12 16:29:58 2023] systemd-journald[1457029]: /var/log/journal/XXXX/system.journal: Journal header limits reached or header out-of-date, rotating.
```
I would also like to silent this. I agree with @plomari: this is really irrelevant to me.

**Comment by guibracco** [comment]
I have the same problem as @manio. On my Raspberry Pi — running Raspberry Pi OS (64-bit, Debian Bookworm) and `systemd 252 (252.19-1~deb12u1)` — , dmesg is flooded with messages by `systemd-journald` like these: 
```
[540026.575375] systemd-journald[288]: Data hash table of /var/log/journal/xxxxxxxxxx/system.journal has a fill level at 75.0 (174764 of 233016 items, 50331648 file size, 287 bytes per hash table item), suggesting rotation.
[540026.575430] systemd-journald[288]: /var/log/journal/xxxxxxxxxx/system.journal: Journal header limits reached or header out-of-date, rotating.
```
Unfortunately, other, way more relevant messages get buried in the log. I wish there was a way of stopping that.

**Comment by BiRo70** [comment]
Same here, dmesg flooded with systemd-journald messages.
`OS: Armbian (24.2.1) armv7l bookworm
Host: Hardkernel ODROID XU4 
Kernel: 6.1.79-current-odroidxu4`

**Comment by yuwata** [comment]
Already fixed by afc47ee2af456d12670df862457dcc7f6b864d79. Closing.
